src/jira/client/jira-client.test.ts

```diff
@@ -144,12 +144,12 @@ describe("Test getting a jira client", () => {
 
 	describe("Reading encryptedSharedSecret", () => {
 		beforeEach(async ()=>{
-			const inst: Installation = await Installation.findOne({
+			const inst: Installation | null = await Installation.findOne({
```

Comment:
In our Installation model, when we call the findOne method we define the return type as `Promise<Installation | null>`.